src/workerd/io/trace.c++

```diff
@@ -260,13 +260,15 @@ Trace::Trace(kj::Maybe<kj::String> stableId,
     kj::Maybe<kj::String> scriptName,
     kj::Maybe<kj::Own<ScriptVersion::Reader>> scriptVersion,
     kj::Maybe<kj::String> dispatchNamespace,
+    bool isActor,
```

Comment:
Really not a fan of boolean arguments like this. Would prefer an enum.

Also, would prefer adding this to the end of the arg list.

Reply:
Hmm, in fact, perhaps the API we expose should be an enum instead of a boolean... like:

```
executionModel: "stateless" | "durableObject" | "workflow";
```

Reply:
Changing to an enum is something I can do. What is the meaning of stateless and workflow in this context?

Reply:
"stateless" would be a regular worker (i.e. not a DO). "workflow" would be the new Workflows product (not launched yet). Workflows are a different execution model from both regular workers and DOs. They are more like "Durable Procedures". cc @sidharthachatterjee